src/transports/ssl.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2024 Tether Operations Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+'use strict'
+
+import MempoolElectrumClient from './client/mempool-electrum-client.js'
+
+/** @typedef {import('./client/mempool-electrum-client.js').MempoolElectrumConfig} MempoolElectrumConfig */
```

Comment:
shouldn't this be:
```suggestion
/** @typedef {import('./client/mempool-electrum-client.js').MempoolClientConfig} MempoolClientConfig */
```
Check other transport files.